public/client.js

```
@@ -0,0 +1,534 @@
+            errorOutput.value = 'Band range cannot be larger than max frequency';
+            return;
+        }
+        
+        if (bandRange < (maxFrequency - minFrequency) / 100) {
+            errorOutput.value = 'Band range is too small for the given frequency range';
+            return;
+        }
+        
+        // Validate frequencies and amplitudes based on waveform type
+        if (waveformTypeSelect.value === 'sine') {
+            const frequency = parseFloat(document.getElementById('frequency').value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)